src/SFA.DAS.ApplyService.Database/Script.PostDeployment1.sql

```diff
@@ -11,95 +11,7 @@ Post-Deployment Script Template
--- 	VALUES (NEWID(), 'Live', 'ApplyEPAOAlertSubmission', 'a56c47c8-6310-4f5c-a3f6-9e996c375557', 'Post16Non-CollegeFHA@Educationgovuk.onmicrosoft.com', GETDATE(), 'System')
--- END
--- GO
--- 
--- IF NOT EXISTS( SELECT * FROM [dbo].[EmailTemplates] WHERE [TemplateId] = 'e0a52c44-10be-4164-9543-3c312769c4e3')
--- BEGIN
--- 	INSERT INTO [dbo].[EmailTemplates] ([Id], [Status], [TemplateName], [TemplateId], [CreatedAt], [CreatedBy])
--- 	VALUES (NEWID(), 'Live', 'ApplyEPAOStandardSubmission', 'e0a52c44-10be-4164-9543-3c312769c4e3', GETDATE(), 'System')
--- END
--- GO
--- 
--- IF NOT EXISTS( SELECT * FROM [dbo].[EmailTemplates] WHERE [TemplateId] = '5bb920f4-06ec-43c7-b00a-8fad33ce8066')
--- BEGIN
--- 	INSERT INTO [dbo].[EmailTemplates] ([Id], [Status], [TemplateName], [TemplateId], [CreatedAt], [CreatedBy])
--- 	VALUES (NEWID(), 'Live', 'EPAOUserApproveRequest', '5bb920f4-06ec-43c7-b00a-8fad33ce8066', GETDATE(), 'System')
--- END
--- GO
--- 
--- IF NOT EXISTS( SELECT * FROM [dbo].[EmailTemplates] WHERE [TemplateId] = '539204f8-e99a-4efa-9d1f-d0e58b26dd7b')
--- BEGIN
--- 	INSERT INTO [dbo].[EmailTemplates] ([Id], [Status], [TemplateName], [TemplateId], [CreatedAt], [CreatedBy])
--- 	VALUES (NEWID(), 'Live', 'EPAOUserApproveConfirm', '539204f8-e99a-4efa-9d1f-d0e58b26dd7b', GETDATE(), 'System')
--- END
--- GO
---
-
--- AB 02/04/19 When coding has been done will need to swapout the templateIds for ApplyEPAOUpdate & ApplyEPAOResponse
--- Was acf63bea-41ff-4a45-a376-9d557c30bca0 
-IF EXISTS (SELECT * FROM EmailTemplates WHERE TemplateName = N'ApplyEPAOUpdate' AND TemplateId = 'acf63bea-41ff-4a45-a376-9d557c30bca0')
-BEGIN
-UPDATE EmailTemplates SET TemplateId =  'ffe63c0d-b2b0-461f-b99a-73105d7d5fa3', UpdatedAt = GETDATE(), UpdatedBy ='System'
-WHERE TemplateName = N'ApplyEPAOUpdate'
-END
-
-IF NOT EXISTS (SELECT * FROM EmailTemplates WHERE TemplateName = N'ApplyEPAOUpdate')
-BEGIN
-INSERT EmailTemplates ([Id], [Status], [TemplateName],[TemplateId],[CreatedAt],[CreatedBy]) 
-VALUES (NEWID(), 'Live', N'ApplyEPAOUpdate', 'ffe63c0d-b2b0-461f-b99a-73105d7d5fa3', GETDATE(), 'System')
-END
-
--- was 2ebc498c-2544-42db-b73e-a6c381c614df
-IF EXISTS (SELECT * FROM EmailTemplates WHERE TemplateName = N'ApplyEPAOResponse' AND TemplateId = '2ebc498c-2544-42db-b73e-a6c381c614df')
-BEGIN
-UPDATE EmailTemplates SET TemplateId =  N'84174eab-f3c1-4274-8670-2fb5b21cbd77', UpdatedAt = GETDATE(), UpdatedBy ='System'
-WHERE TemplateName = N'ApplyEPAOResponse'
-END
-
-IF NOT EXISTS (SELECT * FROM EmailTemplates WHERE TemplateName = N'ApplyEPAOResponse')
-BEGIN
-INSERT EmailTemplates ([Id], [Status],[TemplateName],[TemplateId],[CreatedAt],[CreatedBy]) 
-VALUES (NEWID(), 'Live', N'ApplyEPAOResponse', N'84174eab-f3c1-4274-8670-2fb5b21cbd77', GETDATE(), 'System')  
-END
-
-DELETE FROM EmailTemplates
-WHERE [TemplateName] IN ( 'EPAOUserApproveRequest' , 'EPAOUserApproveConfirm' )
+DELETE FROM [EmailTemplates]
```

Comment:
Are we sure we should clearing out all of the EmailTemplates?

Reply:
This is from PR
https://github.com/SkillsFundingAgency/das-apply-service/pull/421